mmdet/models/roi_heads/bbox_heads/convfc_bbox_head.py

```diff
@@ -170,7 +174,10 @@ def forward(self, x):
 
         cls_score = self.fc_cls(x_cls) if self.with_cls else None
         bbox_pred = self.fc_reg(x_reg) if self.with_reg else None
-        return cls_score, bbox_pred
+        if return_feat:
+            return cls_score, bbox_pred, feature
+        else:
+            return cls_score, bbox_pred
```

Comment:
I suggest refactoring this function into two parts, `forward_shared`, `forward_cls_reg`. So we don't need `return_feat` argument.

Reply:
I agree that using two functions will be more clear. However, ``return_shared_feat`` at the common ``forward()`` should be required.